lib/alephant/broker/cache.rb

```diff
@@ -1,35 +1,33 @@
-require 'dalli-elasticache'
-require 'alephant/logger'
+require "dalli-elasticache"
+require "alephant/logger"
 
 module Alephant
   module Broker
     module Cache
       class Client
         include Logger
 
-        DEFAULT_TTL  = 2592000
+        DEFAULT_TTL = 2_592_000
```

Comment:
Yeah, underscores are ignored in Ruby integers - makes reading large numbers a little easier.